projects/1.19/assets/sweetmagic/sweetmagic/lang/zh_cn.json

```
@@ -0,0 +1,1097 @@
+	"block.sweetmagic.wand_pedastal": "法杖基座",
+	"block.sweetmagic.corkboard": "软木展示框",
+	"block.sweetmagic.wallboard": "壁挂展示框",
+	"block.sweetmagic.wallboard_b": "壁挂展示框（黑）",
+	"block.sweetmagic.couter_table_sink_white_brick_r": "白线砖厨房水槽（红）",
+	"block.sweetmagic.iris_creation": "魔力坩埚",
+	"block.sweetmagic.mill": "磨粉机",
+	"block.sweetmagic.oven": "烤箱",
+	"block.sweetmagic.frypan_r": "平底锅（红）",
+	"block.sweetmagic.pot_w": "锅（白）",
+	"block.sweetmagic.bottle": "发酵瓶",
+	"block.sweetmagic.stove": "灶台",
+	"block.sweetmagic.freezer": "冰箱",
+	"block.sweetmagic.freezer_top": "冰箱",
+	"block.sweetmagic.juice_maker": "饮品机",
+	"block.sweetmagic.magiclight": "魔力光源",
+	"block.sweetmagic.twilightlight": "暮光",
+	"block.sweetmagic.glow_light": "荧光灯",
+	"block.sweetmagic.glow_light_o": "荧光灯（橙）",
+	"block.sweetmagic.gorgeous_lamp": "精致荧光灯",
+	"block.sweetmagic.sweetpotato_plant": "红薯植株",
+	"block.sweetmagic.strawberry_plant": "草莓植株",
+	"block.sweetmagic.onion_plant": "洋葱植株",
+	"block.sweetmagic.olive_plant": "橄榄植株",
+	"block.sweetmagic.soybean_plant": "大豆植株",
+	"block.sweetmagic.rice_plant": "水稻植株",
+	"block.sweetmagic.coffee_plant": "咖啡植株",
+	"block.sweetmagic.corn_plant": "玉米植株",
+	"block.sweetmagic.eggplant_plant": "茄子植株",
+	"block.sweetmagic.tomato_plant": "番茄植株",
+	"block.sweetmagic.lettuce_plant": "生菜植株",
+	"block.sweetmagic.cabbage_plant": "卷心菜植株",
+	"block.sweetmagic.mint_plant": "纸薄荷植株",
+	"block.sweetmagic.cornflower": "魔力矢车菊",
+	"block.sweetmagic.lily_valley": "魔力铃兰",
+	"block.sweetmagic.cosmos": "波斯菊",
+	"block.sweetmagic.blackrose": "黑玫瑰",
+	"block.sweetmagic.white_clover": "白苜蓿",
+	"block.sweetmagic.foxtail_grass": "狐尾草",
+	"block.sweetmagic.snowdrop": "雪莲花",
+	"block.sweetmagic.turkey_balloonflower": "洋桔梗",
+	"block.sweetmagic.iberis_umbellata": "珍珠球",
+	"block.sweetmagic.ultramarine_rose": "青木槿",
+	"block.sweetmagic.solid_star": "满天星",
+	"block.sweetmagic.zinnia": "百日菊",
+	"block.sweetmagic.campanula": "风铃草",
+	"block.sweetmagic.primula_polyansa": "多花报春",
+	"block.sweetmagic.hydrangea": "绣球花",
+	"block.sweetmagic.carnation_crayola": "康乃馨",
+	"block.sweetmagic.christmas_rose": "冬蔷薇",
+	"block.sweetmagic.portulaca": "马齿苋",
+	"block.sweetmagic.surfinia": "矮牵牛",
+	"block.sweetmagic.pansy_blue": "三色堇（蓝）",
+	"block.sweetmagic.pansy_yellowmazenta": "三色堇（黄）",
+	"block.sweetmagic.marigold": "金盏花",
+	"block.sweetmagic.christmarose_ericsmithii": "圣诞玫瑰",
+	"block.sweetmagic.drizzly_mysotis_pot": "细雨勿忘我盆栽",
+	"block.sweetmagic.twilightalstroemeria_pot": "暮光六出盆栽",
+	"block.sweetmagic.snowdrop_pot": "雪莲花盆栽",
+	"block.sweetmagic.turkey_balloonflower_pot": "洋桔梗盆栽",
+	"block.sweetmagic.ultramarine_rose_pot": "青木槿盆栽",
+	"block.sweetmagic.solid_star_pot": "满天星盆栽",
+	"block.sweetmagic.zinnia_pot": "百日菊盆栽",
+	"block.sweetmagic.hydrangea_pot": "绣球花盆栽",
+	"block.sweetmagic.christmarose_ericsmithii_pot": "圣诞玫瑰盆栽",
+	"block.sweetmagic.cosmos_pot": "波斯菊盆栽",
+	"block.sweetmagic.warp_block": "传送祭坛",
+	"block.sweetmagic.mf_fisher": "魔力捕鱼机",
+	"block.sweetmagic.mf_forer": "魔力屠宰场",
```

```suggestion
	"block.sweetmagic.mf_forer": "魔力屠宰机",
```
原文是拉丁语`forer` = 英文`bore`/`pierce`